.github/workflows/smoke-test-package-bypass.yaml

```diff
@@ -0,0 +1,38 @@
+# This workflow is required to ensure that required GitHub check passes even if
+# the actual "Smoke test package rpm/deb" workflow skipped due to path filtering. Otherwise,
+# it will stay forever pending.
+#
+# See "Handling skipped but required checks" for more info:
+#
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+#
+# Note both workflows must have the same name.
+
+name: Smoke test package rpm/deb
+run-name: Smoke test package rpm/deb
+
+on:
+  pull_request:
+  merge_group:
+    paths-ignore:
+      - '.github/workflows/smoke-test-package.yaml'
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - '**.rs'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
```

Comment:
This is syntactically invalid and does nothing. The workflows need to be combined with a "bypass check" step added.